tests/std/tests/P0019R8_atomic_ref/test.cpp

```diff
@@ -269,6 +269,26 @@ void test_ptr_ops() {
     assert(ry.load() == a + 0x10);
 }
 
+// GH-1497 <atomic>: atomic_ref<const T> fails to compile
+void test_gh_1497() {
+    {
+        const int ci{1729};

```

Comment:
We should test with a truly-`const` object of static storage duration that will be stored in read-only memory to ensure the class really doesn't perform writes.

Reply:
I had forgotten about that particular horror, thanks. I'll make this `static constexpr` with a comment.